R/correctionsataglance-data.R

```diff
@@ -223,6 +227,11 @@ findOverlap <- function(dataList){
         dataIn$line_num <- 1
         
         #ordered by applied date for process order data
+        if (isEmptyOrBlank(dataIn$applyDates)) {
+          for (i in 1:length(dataIn)) {
```

Comment:
You shouldn't need to have the for loop. Just making `dataIn$applyDates <- NA` will work because data.frame will automatically fill in the other NAs. @mhines-usgs I'll go ahead and make a fix for this since you are traveling!

Example:
```r
dataIn <- list(values = 1:10, dates = 1:10, applyDates = NULL)
as.data.frame(dataIn)

Error in data.frame(values = 1:10, dates = 1:10, applyDates = NULL, check.names = TRUE,  : 
  arguments imply differing number of rows: 10, 0

dataIn <- list(values = 1:10, dates = 1:10, applyDates = NA)
as.data.frame(dataIn)
   values dates applyDates
1       1     1         NA
2       2     2         NA
3       3     3         NA
4       4     4         NA
5       5     5         NA
6       6     6         NA
7       7     7         NA
8       8     8         NA
9       9     9         NA
10     10    10         NA
```